src/main/webapp/app/exercises/shared/exercise-scores/manage-assessment-buttons.component.html

```diff
@@ -0,0 +1,60 @@
+        <div
+            *ngIf="
+                (correctionRound == 0 || participation.results?.[correctionRound - 1]?.completionDate) &&
+                (newManualResultAllowed ||
+                    (participation.results?.[correctionRound]?.assessmentType && participation.results?.[correctionRound]?.assessmentType !== AssessmentType.AUTOMATIC))
+            "
+        >
+            <a
+                [routerLink]="getAssessmentLink(correctionRound)"
+                [queryParams]="{ 'correction-round': getCorrectionRoundForAssessmentLink(correctionRound) }"
+                class="btn btn-sm me-1 mb-2"
+                [class.btn-success]="
+                    (!participation.results?.[correctionRound]?.assessmentType || participation.results?.[correctionRound]?.assessmentType === AssessmentType.AUTOMATIC) &&
+                    !participation.results?.[correctionRound]?.hasComplaint
+                "
+                [class.btn-primary]="participation.results?.[correctionRound]?.completionDate || participation.results?.[correctionRound]?.hasComplaint"
+                [class.btn-warning]="
+                    participation.results?.[correctionRound] && !participation.results?.[correctionRound]?.completionDate && !participation.results?.[correctionRound]?.hasComplaint
+                "
+            >
+                <fa-icon [icon]="faFolderOpen" [fixedWidth]="true"></fa-icon>
+                <ng-container *ngIf="!participation.results?.[correctionRound]?.hasComplaint; else complaint">
+                    {{
+                        'artemisApp.assessment.dashboard.actions.' +
+                            (this.examMode ? 'examCorrectionRound.' : '') +
+                            (!participation.results?.[correctionRound]?.assessmentType || participation.results?.[correctionRound]?.assessmentType === AssessmentType.AUTOMATIC
+                                ? 'assess'
+                                : participation.results?.[correctionRound]?.completionDate
+                                  ? 'open'
+                                  : 'continue') | artemisTranslate: { correctionRound: correctionRound + 1 }
+                    }}
+                </ng-container>
+                <ng-template #complaint>
+                    {{ 'artemisApp.exerciseAssessmentDashboard.showComplaint' | artemisTranslate }}
+                </ng-template>
+            </a>
+            <button
+                *ngIf="
+                    newManualResultAllowed &&
+                    participation.results?.[correctionRound]?.assessmentType &&
+                    !participation.results?.[correctionRound]?.completionDate &&
+                    participation.results?.[correctionRound]?.assessmentType !== AssessmentType.AUTOMATIC
+                "
+                (click)="cancelAssessment(participation.results?.[correctionRound]!, participation)"
+                [disabled]="isLoading"
+                class="btn btn-danger btn-sm mb-1"
```

Comment:
Noticed that the _cancel assessment_ button is misaligned compared to the _continue assessment_ button.
It was not introduced by this PR, but should be fixed anyway:
![image](https://github.com/ls1intum/Artemis/assets/9954674/bb5b6a86-d936-41cd-b47a-5b76b13b1e1c)

```suggestion
                class="btn btn-sm me-1 mb-2 btn-danger"
```